analyzer/build.gradle.kts

```diff
@@ -73,6 +73,11 @@ dependencies {
     implementation(libs.semver4j)
     implementation(libs.sw360Client)
     implementation(libs.toml4j)
+    constraints {
```

Comment:
Nit: Maybe also add an empty line before line 75 to more clearly group the constraint to the direct dependency that introduces this transitive dependency.